lib/email/sender.rb

```diff
@@ -360,6 +360,8 @@ def add_attachments(*posts)
 
       email_size = 0
       posts.each do |post|
+        next unless DiscoursePluginRegistry.apply_modifier(:should_add_attachments, post)
```

Comment:
would `should_add_email_attachments` be a more descriptive name? (so it's clear we're not talking about chat message attachments or anything else)

Reply:
Sure thing 👍